tests/stdlib/test_math.py

```diff
@@ -9,3 +9,29 @@ def test_sqrt(self):
             import math
             print(math.sqrt(1))
             """)
+
+    @expectedFailure
+    def test_neg_sqrt(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt(-1))
+            """)
+
+    @expectedFailure
+    def test_no_args(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt())
+            """)
+
+    @expectedFailure
+    def test_too_many_args(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt(1,2))
+            """)
+
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt(1,2,3,4,5))
+            """)
```

Comment:
Expected empty new line at end
```suggestion
            """)

```